src/preprocessing/passes/non_clausal_simp.cpp

```diff
@@ -114,8 +114,8 @@ PreprocessingPassResult NonClausalSimp::applyInternal(
       << "Iterate through " << propagator->getLearnedLiterals().size()
       << " learned literals." << std::endl;
   // No conflict, go through the literals and solve them
-  SubstitutionMap& top_level_substs =
-      d_preprocContext->getTopLevelSubstitutions();
+  TrustSubstitutionMap& ttls = d_preprocContext->getTopLevelSubstitutions();
+  CVC4_UNUSED SubstitutionMap& top_level_substs = ttls.get();
```

Comment:
Why `CVC4_UNUSED`?

Reply:
The only place top_level_substs is used on proof-new is within debug assertions. Marking it as CVC4_UNUSED ensures that we don't give a warning on production.